test/test-eth-trezor-keyring.js

```diff
@@ -694,4 +694,70 @@ describe('TrezorKeyring', function () {
       }
     });
   });
+
+  describe('getVendor', function () {
+    it('should call TrezorConnect.getFeatures if we dont have vendor name', async function () {
+      sinon
+        .stub(TrezorConnect, 'getFeatures')
+        .callsFake(() => Promise.resolve({}));
+
+      await keyring.getVendor();
+      assert(TrezorConnect.getFeatures.calledOnce);
+    });
+
+    it('should return null when getFeatures not success', async function () {
```

Comment:
This sounds like `TrezorConnect.getFeatures` fails, but it doesn't. What do you think about this?

``` suggestion
    it('should return null when TrezorConnect.getFeatures resolves with an object that does not have a success property', async function () {
```